docker/Dockerfile

```diff
@@ -9,8 +9,8 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-#@follow_tag(registry.redhat.io/rhel9/go-toolset:1.19)
-FROM registry.access.redhat.com/ubi9/go-toolset:1.19.13-4.1697647145 as builder
+#@follow_tag(registry.redhat.io/rhel9/go-toolset:latest)
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10-6 as builder
```

Comment:
go.mod should be updated to use the same version of go.

Reply:
if you want, sure. I think a go mod 1.19 will build in a 1.20 env.

Reply:
https://github.com/janus-idp/operator/pull/82